understanding/20/use-of-color.html

```diff
@@ -38,13 +38,28 @@ <h2>Intent of Use of Color</h2>
          only in color.
       </p>
       
-      <p>Examples of information conveyed by color differences: “required fields are red",
-         “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples
-         of indications of an action include: using color to indicate that a link will open
-         in a new window or that a database entry has been updated successfully. An example
-         of prompting a response would be: using highlighting on form fields to indicate that
-         a required field had been left blank.
-      </p>
+      <p>Examples of common failures include:</p>
+      <ul>
+        <li>
+            <strong>Information conveyed by color differences:</strong> 
+            <ul>
+                <li>required fields are only identified by changing their border to a red color;</li>
+                <li>the type of transaction is only visually distinguished by using different foreground colors: red for debits and green for credits;</li>
+                <li>subway lines are visually distinguished solely by employing various filled colors.</li>
+            </ul>
+        </li>
+        <li>using color alone to indicate
+         that a link will open in a new window or that a database entry has been updated successfully.</li>
+        <li>using subtle color highlighting on form fields to prompt for a response on 
+         required fields that have been left blank.</li>
+        <li><strong>Distinguishing a visual element:</strong>
+            <ul>
+                <li>links are only distinguished from surrounding text using a subtle color difference (with a contrast difference that is less than 3:1);</li>
+                <li>for a series of toggle buttons, the pressed and unpressed buttons are only visually indicated by a subtle difference in color;</li>
+                <li>the interactive element that currently has focus is only distinguished by changing the color of the element's existing background or border.</li>
```

Comment:
I am removing this example. I believe that including indicators of focus in the Use of Color discussion for anything other than text color changes is likely to result in confusion. Focus Visible does not define a minimum contrast variation, and Use of Color does not list state or focus in its wording. Further, some users are definitely going to be confused about whether shading and other additive visual effects are uses of color. These short comings are specifically why Focus Appearance was introduced in 2.2
```suggestion
```

Reply:
> Use of Color does not list state or focus in its wording

doesn't mean, in my view, that it excludes it. adding extra visuals is not a use of color. only changing an existing color to another color (with a change that's less than 3:1) is a failure of use of color, I'd argue, as it exactly fits the definition of "Color is not used as the only visual means of conveying information" (where the information here is the state of the component ... not just focus, but also pressed/toggled/etc)